java-type-checker/java_type_checker/expressions.py

```diff
@@ -88,6 +109,30 @@ def __init__(self, receiver, method_name, *args):
         self.method_name = method_name
         self.args = args
 
+    def static_type(self):
+        return self.receiver.static_type().method_named(self.method_name).return_type
+
+    def check_types(self):
+        self.receiver.check_types()
+        for arg in self.args:
+            arg.check_types()
+
+        self.receiver.static_type().method_named(self.method_name)
```

Comment:
This line does nothing. I put it in while figuring things out and forgot to remove it.